### PR TITLE
DE 1573 Creating new report

### PR DESCRIPTION
**Pull request by k8tied1**

### CODE REVIEW

File: CI/SQL/20160705_161700_DE1573-createReport_Event_Double_Booked_CRDS.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+           ([Report_ID]
+		   ,[Report_Name]
+           ,[Description]
+           ,[Report_Path]
+           ,[Pass_Selected_Records]
+           ,[Pass_LinkTo_Records]
+           ,[On_Reports_Tab])
+     VALUES
+           (287
+		   ,'Double Booked Rooms-CRDS'
```
Comment (charlie-retzler) [inline CI/SQL/20160705_161700_DE1573-createReport_Event_Double_Booked_CRDS.sql]: Remove the CRDS at the end of the Report Name that shows in MP

File: CI/SQL/20160705_163700_DE1573-ReportRoles.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+IF NOT EXISTS ( SELECT * FROM dp_Role_Reports 
+                WHERE  Role_ID = 100
+                   AND Report_ID = 287
+                   AND Domain_ID = 1 ) 
+ BEGIN
+	INSERT INTO [dbo].[dp_Role_Reports]([Role_ID],[Report_ID],[Domain_ID])
+	VALUES (100,287,1 )
+ END
+
```
Comment (charlie-retzler) [inline CI/SQL/20160705_163700_DE1573-ReportRoles.sql]: Should we remove the existing report role?

File: CI/SQL/20160705_161700_DE1573-createReport_Event_Double_Booked_CRDS.sql (hunk 1 above)
Comment (k8tied1) [inline CI/SQL/20160705_161700_DE1573-createReport_Event_Double_Booked_CRDS.sql]: Done

File: CI/SQL/20160705_163700_DE1573-ReportRoles.sql (hunk 2 above)
Comment (k8tied1) [inline CI/SQL/20160705_163700_DE1573-ReportRoles.sql]: Done - new file added

File: CI/SQL/20160705_163700_DE1573-ReportRoles.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+USE [MinistryPlatform]
+GO
+
+IF NOT EXISTS ( SELECT * FROM dp_Role_Reports 
+                WHERE  Role_ID = 100
+                   AND Report_ID = 287
+                   AND Domain_ID = 1 ) 
+ BEGIN
+	INSERT INTO [dbo].[dp_Role_Reports]([Role_ID],[Report_ID],[Domain_ID])
+	VALUES (100,287,1 )
```
Comment (charlie-retzler) [inline CI/SQL/20160705_163700_DE1573-ReportRoles.sql]: Should this be all the existing roles that were on ReportID 213?

**Pull request merged**